### DISCUSSION TITLE
node节点的labelText是否支持换行或更改fontSize,节点文本过长时只能省略吗

### DISCUSSION THREAD

**Discussion by canbaoSama** [antvis/G6#6240] [Q&A]
node节点的labelText是否支持换行或更改fontSize,节点文本过长时只能省略吗

**Comment by resetsix** [comment]
> node节点的labelText是否支持换行或更改fontSize,节点文本过长时只能省略吗

修改label字体大小参考：https://github.com/antvis/G6/discussions/6204#discussioncomment-10543604

设置文本超出省略：https://g6-next.antv.antgroup.com/manual/faq#%E8%AE%BE%E7%BD%AE%E6%96%87%E6%9C%AC%E8%B6%85%E5%87%BA%E7%9C%81%E7%95%A5

**Reply by canbaoSama** [reply]
其实我想问的是设置3行省略这样子的功能

**Reply by resetsix** [reply]
> 其实我想问的是设置3行省略这样子的功能

<details>
<summary>点击展开/关闭</summary>

<img width="822" alt="图片" src="https://github.com/user-attachments/assets/c5001985-270e-451b-a624-471db3d50eb3">

</details>

参考：https://g6-next.antv.antgroup.com/examples/element/label#word-wrap

**Reply by resetsix** [reply]
> 其实我想问的是设置3行省略这样子的功能

看到了吗
